holoviews/plotting/plotly/element.py

```diff
@@ -603,11 +603,19 @@ def get_color_opts(self, eldim, element, ranges, style):
         opts = {}
         dim_name = dim_range_key(eldim)
         if self.colorbar:
-            if isinstance(eldim, dim):
-                title = str(eldim) if eldim.ops else str(eldim)[1:-1]
-            else:
-                title = eldim.pprint_label
-            opts['colorbar'] = dict(title=title, **self.colorbar_opts)
+            opts['colorbar'] = dict(**self.colorbar_opts)
+            if not "title" in opts['colorbar']:
+                if isinstance(eldim, dim):
+                    title = str(eldim)
+                    if eldim.ops:
+                        title = title
+                    elif title.startswith("dim('") and title.endswith("')"):
+                        title = title[5:-2]
+                    else:
+                        title = title[1:-1]
```

Comment:
I don't know why title[1:-1] is a valid value from time to time.

Reply:
Properly because of something like this `title = "'An awesome title'"`

Reply:
That's pretty strange!